third_party/subscriptions-project/config.js

```diff
@@ -590,7 +590,7 @@ function pushIfNonEmpty(array, val) {
 function toString(val) {
   // Do check equivalent to `val instanceof Element` without cross-window bug
   if (val && val.nodeType == 1) {
-    return val.tagName.toLowerCase() + (val.id ? '#' + val.id : '');
+    return val.localName + (val.id ? '#' + val.id : '');
```

Comment:
I don't think we should be changing third_party code.

Reply:
Done.